.github/workflows/pull-request.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
             arch: arm64
           - os: windows
             run: ubuntu-24.04
-            targets: ci-go-ci-build-windows
+            targets: ci-build-windows
```

Comment:
ℹ️ We'll no longer build the windows binaries in the golang container image: We need the zig compiler stuff and that's easier to procure in a plain linux runner environment.